Formula/d/dovitool.rb

```diff
@@ -0,0 +1,19 @@
+class Dovitool < Formula
+  desc "CLI tool for Dolby Vision metadata on video streams"
+  homepage "https://github.com/quietvoid/dovi_tool/"
+  url "https://github.com/quietvoid/dovi_tool/archive/refs/tags/2.0.3.tar.gz"
+  sha256 "6a4afe2733ab1a4eca6ff98c6038f60bbbed5c649b99c239759ccf0b17a3f818"
+  license "MIT"
+  head "https://github.com/quietvoid/dovi_tool.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    assert_match "CLI tool combining multiple utilities for working with Dolby Vision",
```

Comment:
need a better test to test the tool functionality

Reply:
I'm attempting to write another test here, but I'm having trouble accessing the files extracted from the source code archive.  There's a file in there with a relative path of `assets/tests/fel_orig.bin` that I'm trying to reference, and I keep getting a missing file error.  I thought I'd be able to use `#{buildpath}` to reference it, but that doesn't seem to have content in a test block, and the contents of the source archive don't appear to be available in `#{testpath}`.

What would be the best way of referencing this file?